README.md

```diff
@@ -1,52 +1,74 @@
-# Tendermint Socket Protocol (ABCI)
+# Application BlockChain Interface (ABCI)
 
 [![CircleCI](https://circleci.com/gh/tendermint/abci.svg?style=svg)](https://circleci.com/gh/tendermint/abci)
 
-Blockchains are a system for creating shared multi-master application state. 
-**ABCI** is a socket protocol enabling a blockchain consensus engine, running in one process,
-to manage a blockchain application state, running in another.
+Blockchains are a system for multi-master state machine replication. 
+**ABCI** is an interface that defines the boundary between the replication engine (the blockchain),
+and the state machine (the application). It is most commonly implemented in the form of the Tendermint Socket Protocol (TSP),
+also known as Teaspoon.
+By using a socket protocol, we enable a consensus engine running in one process
+to manage an application state running in another.
 
-For more information on ABCI, motivations, and tutorials, please visit [our blog post](https://tendermint.com/blog/abci-the-tendermint-socket-protocol).
+For more information on ABCI, motivations, and tutorials, please visit [our blog post](https://tendermint.com/blog/tmsp-the-tendermint-socket-protocol).
```

Comment:
broken link. https://tendermint.com/blog/abci-the-application-blockchain-interface ?